Directory.Build.props

```diff
@@ -227,8 +228,7 @@
     <MicrosoftNetCoreIlasmPackageRuntimeId Condition="'$(MicrosoftNetCoreIlasmPackageRuntimeId)' == ''">$(ToolRuntimeRID)</MicrosoftNetCoreIlasmPackageRuntimeId>
 
     <!-- support cross-targeting by choosing a RID to restore when running on a different machine that what we're build for -->
-    <_portableOS Condition="'$(OSGroup)' == 'Unix' AND '$(_runtimeOSFamily)' != 'osx' AND '$(_runtimeOSFamily)' != 'FreeBSD'">linux</_portableOS>
-    <_portableOS Condition="'$(OSGroup)' == 'Windows_NT'">win</_portableOS>
```

Comment:
Are you removing these because it will be set by the _runtimeOSFamily above? Could you please make sure that it sets _portableOS correctly in your local box with removing this line?